table/server/underdb/glue/src/main/java/alluxio/table/under/glue/GlueUtils.java

```diff
@@ -0,0 +1,161 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.table.under.glue;
+
+import alluxio.grpc.table.Schema;
+import alluxio.grpc.table.layout.hive.HiveBucketProperty;
+import alluxio.grpc.table.layout.hive.SortingColumn;
+import alluxio.grpc.table.layout.hive.Storage;
+import alluxio.grpc.table.layout.hive.StorageFormat;
+import alluxio.table.common.udb.PathTranslator;
+
+import com.amazonaws.services.glue.model.Column;
+import com.amazonaws.services.glue.model.Order;
+import com.amazonaws.services.glue.model.StorageDescriptor;
+import org.apache.hadoop.hive.common.FileUtils;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+/**
+ * Glue utils.
+ */
+public class GlueUtils {
```

Comment:
For all of these conversions, does it handle when fields don't exist or are `null`? We have hit several issues with the hive one because of these scenarios.

Reply:
I have fixed some of the `null` filed issues in `glueUtils`, I will add more UT for these scenarios.